packages/manager/apps/web/client/app/domain-operation/domain-operation.html

```diff
@@ -109,23 +109,18 @@ <h1 data-translate="domains_operations_dashboard_title"></h1>
                     <td data-ng-bind="::operation.doneDate | date:'short'"></td>
                     <td class="text-nowrap" data-ng-bind="(('domains_operations_statusOperation_' + operation.status | translate) == 'domains_operations_statusOperation_' + operation.status)? operation.status: ('domains_operations_statusOperation_' + operation.status | translate)" title="{{operation.comment}}"></td>
                     <td class="text-center">
-                        <span data-uib-tooltip="{{ 'domains_operations_no_action' | translate }}"
```

Comment:
I guess this popover is not valid anymore ?

Reply:
This says that "No action available for this operation" on disabled `action-menu `in a table. I think disabled `action-menu` itself gives the same meaning so removed it.

Reply:
Good thinking

Reply:
I’m not sure of that, it should be discussed with CX team ;)

I think it’s better to avoid updating this behavior in this PR. We can challenge that, but in another PR. It’s not the goal of this one ;)

Reply:
Have added this back